### DISCUSSION TITLE
[Contributing] First pull request issues

### DISCUSSION THREAD

**Discussion by mariojsnunes** [OrchardCMS/OrchardCore#7366] [Q&A] [answered]
Hey guys,

I'm trying to make my first pull request.
I've already cloned the project locally, created a branch from dev and made my code changes.

I'm getting this error when I try to push my branch:
> Pushing mn/smtp-attachments
> Error encountered while pushing branch to the remote repository: Git failed with a fatal error.
> unable to access 'https://github.com/OrchardCMS/OrchardCore.git/': The requested URL returned error: 403

Do you need to give permissions to my account?

I've checked here and couldn't find info on this.
https://github.com/OrchardCMS/OrchardCore/blob/dev/CONTRIBUTING.md
(The "Look at the documentation" link is broken)

Thanks for the help :)

**Comment by jptissot** [comment, marked as answer]
Hi, you need to fork the orchard repository (in Github) and push the changes to that repository. Then, you will be able to make a pull request from your own forked repository.

**Reply by Skrypt** [reply]
![image](https://user-images.githubusercontent.com/3228637/97047882-9fa23080-1547-11eb-8372-843c157aa558.png)

**Reply by mariojsnunes** [reply]
thanks guys!
Here it is #7380
